test.cfg.example

```diff
@@ -0,0 +1,7 @@
+# To run tests, copy this file to test.cfg and set the values below appropriately.
+
+[cdmtaskservicetest]
+
+# Absolute path to a crane executable
+# See https://github.com/google/go-containerregistry/blob/main/cmd/crane/README.md
```

Comment:
The online installation guide doesn’t provide any details on how to obtain the local path of the executable. Can you add instructions or an example here?

Reply:
It's just a tarball. Do you think we need to add instructions for that?

Reply:
I see. You downloaded the tar file. Hmmm. Maybe just a example string? I did brew install and `which crane` the get the path.

Reply:
> Maybe just a example string?

Not sure what you mean here?

Reply:
something like, `e.g. test.crane.exe=/opt/homebrew/bin/crane`

Reply:
Added some help text

Reply:
👍